lib/eliscore_web/controllers/api/v1/user_controller.ex

```diff
@@ -11,8 +11,8 @@ defmodule EliscoreWeb.UserController do
     |> render("index.json", users: users)
   end
 
-  def show(conn, %{"id" => full_name}) do
-    case Repo.get_by(User, full_name: full_name) do
+  def show(conn, %{"id" => id}) do
+    case Repo.get_by(User, id: id) do
```

Comment:
Repo.get!(User, id) ?

Reply:
`get`, `get!` will result in error insteadd of nil, good catch